.github/workflows/push.yml

```diff
@@ -14,3 +14,22 @@ jobs:
       - uses: release-drafter/release-drafter@b1476f6e6eb133afa41ed8589daba6dc69b4d3f5 # v6
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  notify:
+    if: always() && needs.update_release_draft.result == 'failure' && github.ref == 'refs/heads/main'
+    runs-on: ubuntu-latest
+    needs:
+      - update_release_draft
+    env:
+      SLACK_WEBHOOK_URL: ${{ secrets.DEVTOOLS_CI_SLACK_URL }}
+    steps:
+      - name: Send CI failure notification
+        if: env.SLACK_WEBHOOK_URL != ''
+        run: |
+          MESSAGE_ID="ci-failure-${{ github.run_id }}-$(date +%s)"
+          curl -X POST -H 'Content-type: application/json' \
+          --data "{\"text\":\"❌ Release/CI workflow failed on main branch for '${{ github.repository }}' (push.yml). Check logs: '${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}\", \"ts\":\"$MESSAGE_ID\"}" \
```

Comment:
Do not use emoji in messages, it clutters slack making it harder to read.

See ![](https://sbarnea.com/ss/Screen-Capture-2025-10-08-11-27-48.png)